### PR TITLE
Two more station beacons

### PR DESCRIPTION
**Pull request by Errant-4**
## About the PR
Adds `Evac` and Security `Checkpoint` station beacon prototypes, so no one ends up mapping these with custom beacons that can't be modified en-masse, if they are ever to be renamed recolored etc

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Devices/station_beacon.yml [hunk 1]
```diff
@@ -226,6 +226,14 @@
     - type: NavMapBeacon
       text: law office
 
+- type: entity
+  parent: DefaultStationBeaconSecurity
+  id: DefaultStationBeaconSecurityCheckpoint
+  suffix: Sec Checkpoint
+  components:
+    - type: NavMapBeacon
+      text: checkpoint
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/station_beacon.yml:235]:
```suggestion
  - type: NavMapBeacon
    text: checkpoint
```

File: Resources/Prototypes/Entities/Objects/Devices/station_beacon.yml [hunk 2]
```diff
@@ -552,6 +560,14 @@
   - type: NavMapBeacon
     text: arrivals
 
+- type: entity
+  parent: DefaultStationBeacon
+  id: DefaultStationBeaconEvac
+  suffix: Evac
+  components:
+    - type: NavMapBeacon
+      text: evac
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Devices/station_beacon.yml:569]:
```suggestion
  - type: NavMapBeacon
    text: evac
```

**Review by EmoGarbage404 — APPROVED** [review]

**Comment by Emisse** [comment]
emo review agree

**Comment by Errant-4** [comment]
Done

**Pull request merged**